asciidoc-confluence-publisher-converter/src/main/java/org/sahli/asciidoc/confluence/publisher/converter/FolderBasedAsciidocPagesStructureProvider.java

```diff
@@ -33,6 +33,7 @@ public class FolderBasedAsciidocPagesStructureProvider implements AsciidocPagesS
 
     private static final String ADOC_FILE_EXTENSION = ".adoc";
     private static final String INCLUDE_FILE_PREFIX = "_";
+    private static final String IGNORE_DOT_FILE_SUFFIX = ".";
```

Comment:
I guess my question is should we make this more specific to .asciidoctorconfig files. So rather that filtering for all dot files which may have a wide effect, we're more specific with the filter clause?

Reply:
Yes, I had the same thought when having a look into your code. This would definitely be the safer bet.

Reply:
Hi @emeraldjava, do you still consider adjusting the filter pattern to be specific on `. asciidoctorconfig` files?